packages/ckeditor5-dev-tests/lib/tasks/runautomatedtests.js

```diff
@@ -78,7 +78,7 @@ function createEntryFile( globPatterns, production ) {
 		let hasFiles = false;
 
 		for ( const resolvedPattern of globPatterns[ singlePattern ] ) {
-			const files = globSync( resolvedPattern ).map( file => file.replace( /\\/g, '/' ) );
+			const files = globSync( resolvedPattern );
```

Comment:
It does not work now, because `glob` returns OS-specific path separators.

```suggestion
			const files = globSync( resolvedPattern ).map( filePath => upath.normalize( filePath ) );
```

Reply:
Fixed in [a8f4de5](https://github.com/ckeditor/ckeditor5-dev/pull/1062/commits/a8f4de57434b4d7ba7a7797428d3804244bb215a)